Aquality.WinAppDriver/tests/Aquality.WinAppDriver.Tests/Elements/TextBoxTests.cs

```diff
@@ -0,0 +1,42 @@
+﻿using Aquality.WinAppDriver.Tests.Applications.Locators;
+using NUnit.Framework;
+
+namespace Aquality.WinAppDriver.Tests.Elements
+{
+    public class TextBoxTests : TestWithApplication
+    {
+        private const string ExpectedValue = "1";
+        private readonly CalculatorWindow CalculatorWindow = new CalculatorWindow();
```

Comment:
Non-static fields should be named starting from the lowercase letter.
Also you don't need the CalculatorWindow here, you use only RightArgumentTextBox - let's store just it

Reply:
done

Reply:
done